packages/assistant/src/RPA/Assistant/flet_client.py

```diff
@@ -0,0 +1,183 @@
+import atexit
+import logging
+import os
+import signal
+from typing import Callable, List, Optional, Union, Literal, Tuple
+
+import flet
+from flet import Control, Page, ScrollMode
+from flet.utils import is_windows
+
+from RPA.Assistant.types import Result, Location
+
+
+def resolve_absolute_position(
+    location: Union[Location, Tuple],
+) -> Tuple[int, int]:
+
+    if location is Location.TopLeft:
+        left_coordinate = 0
+        top_coordinate = 0
+        return (left_coordinate, top_coordinate)
+    elif isinstance(location, tuple):
+        return location
+    else:
+        raise ValueError("Locations other than Center or TopLeft are not yet supported")
+
+
+class FletEvent:
+    target: str
+    name: str
+    data: str
+    control: Control
+    page: Page
+
+
+class FletClient:
+    """Class for wrapping flet operations"""
+
+    def __init__(self) -> None:
+        self._conn = self._preload_flet()
+        self.elements: List[List[Control]] = [[]]
+        self.invisible_elements: List[List[Control]] = [[]]
+        self.results: Result = {}
+        self._pagination = 0
+        self.page: Optional[Page] = None
+        self._fvp = None
```

Comment:
Are the non-underscore props actually public or what is the logic here, seems pretty random

Reply:
sorry, attributes, wrong language

Reply:
resolved in 08b33446703e680bb0ceae9c53629e23cd3eb65b